ambry-server/src/main/java/com/github/ambry/server/AmbryRequests.java

```diff
@@ -508,6 +519,122 @@ public void handleDeleteRequest(NetworkRequest request) throws IOException, Inte
             metrics.deleteBlobTotalTimeInMs, null, null, totalTimeSpent));
   }
 
+  /**
+   * Handles a {@link BatchDeleteRequest} and sends out the response via the {@link RequestResponseChannel}.
+   * @param request the {@link NetworkRequest} that contains the {@link BatchDeleteRequest}.
+   * @throws IOException if there is an I/O error reading from the {@link NetworkRequest}.
+   * @throws InterruptedException if the operation is interrupted.
+   */
+  @Override
+  public void handleBatchDeleteRequest(NetworkRequest request) throws IOException, InterruptedException {
+    logger.info("Batch delete request received.");
```

Comment:
Can we add trace logs in the condition where you are handling partial error scenarios ?